sys-boot/udk/udk-2015-r1.ebuild

```diff
@@ -117,92 +121,70 @@ src_configure() {
-		insinto "${include_dest}${f}"
-		doins "${S}/MdePkg/Include${f}"/*.h
-	done
-	insinto "${include_dest}"
-	doins "${S}/MdePkg/Include/${ARCH}"/*.h
+	insinto "/usr/include/${PN}"
+	doins "${S}/MdePkg/Include/"*.h
+	doins -r "${S}/MdePkg/Include/"{${ARCH}/.,Guid,IndustryStandard,Library,Pi,Ppi,Protocol,Uefi}
 	local hfile
-	find "${S}" -name 'BaseTools' -prune -o -name 'MdePkg' -prune -o \
-		-name 'CryptoPkg' -prune -o -type d -name Include \
-		-exec find {} -maxdepth 0 \; \
-		| while read hfile; do
-		doins -r "${hfile}"/*
-	done
+	while read -d '' -r hfile; do
+		doins -r "${hfile}/."
+	done < <(find "${S}" -name 'BaseTools' -prune -o -name 'MdePkg' -prune -o \
+		-name 'CryptoPkg' -prune -o -type d -name Include -print0)
 
 	dobin "${S}/BaseTools/Source/C/bin/GenFw"
 
 	if use doc; then
 		docinto "html"
 		# Document installation may be very long, so split it and display message
 		for f in "${S}"/doc/*; do
-			ebegin "Installing documentation for $(basename ${f}), please wait"
+			ebegin "Install documentation for ${f##*/}"
 			dodoc -r "${f}"
 			eend $?
 		done
 	fi
 
-	local ex_rebuild_dir
-	local ex_name
-	local ex_build_dir
-	if use examples; then
-		ex_rebuild_dir="${S}/${P}-exemples"
-		for f in "${S}/MdeModulePkg/Application"/*; do
-			ex_name=$(basename "${f}")
-			ebegin "Preparing ${ex_name} example"
-			mkdir -p "${ex_rebuild_dir}/${ex_name}" || die
-			ex_build_dir="${build_dir}/MdeModulePkg/Application"
-			ex_build_dir="${ex_build_dir}/${ex_name}/${ex_name}"
-
-			copySourceFiles "${f}" "${ex_rebuild_dir}/${ex_name}"
-			copySourceFiles "${ex_build_dir}/DEBUG" "${ex_rebuild_dir}/${ex_name}"
-			createMakefile "${ex_rebuild_dir}/${ex_name}/Makefile" \
-				"${ex_name}" "${ex_build_dir}/GNUmakefile"
-
-			tar -C "${ex_rebuild_dir}" -cf "${ex_rebuild_dir}/${ex_name}.tar" \
-				"${ex_name}" || die
-
-			eend $? "Failed to create example file"
-		done
-		docinto "examples"
-		dodoc "${ex_rebuild_dir}"/*.tar
-	fi
-
-# TODO * QA Notice: The following files contain writable and executable sections
-# TODO * !WX --- --- usr/lib64/libBaseLib.a:Thunk16.obj
-# TODO * !WX --- --- usr/lib64/libBaseLib.a:SwitchStack.obj
-# TODO * !WX --- --- usr/lib64/libBaseLib.a:SetJump.obj
-# TODO * !WX --- --- usr/lib64/libBaseLib.a:LongJump.obj
-# TODO * !WX --- --- usr/lib64/libBaseLib.a:EnableDisableInterrupts.obj
-# TODO * !WX --- --- usr/lib64/libBaseLib.a:DisablePaging64.obj
-# TODO * !WX --- --- usr/lib64/libBaseLib.a:CpuId.obj
-# TODO * !WX --- --- usr/lib64/libBaseLib.a:CpuIdEx.obj
-# TODO * !WX --- --- usr/lib64/libBaseLib.a:EnableCache.obj
-# TODO * !WX --- --- usr/lib64/libBaseLib.a:DisableCache.obj
-# TODO * QA Notice: Package triggers severe warnings which indicate that it
-# TODO *            may exhibit random runtime failures.
-# TODO * /usr/include/bits/string3.h:90:70: warning: call to void* __builtin___memset_chk(void*, int, long unsigned int, long unsigned int) will always overflow destination buffer
+	#local ex_rebuild_dir
```

Comment:
Why is this commented out?

Reply:
Then please remove the code entirely. There's really no point in having stuff commented out if it's not going to be addressed.

Reply:
OK